src/locale/ngage/SDL_syslocale.cpp

```diff
@@ -0,0 +1,300 @@
+            break;
+        case ELangGreek:
+        case ELangCyprusGreek:
+            locale = "el_GR";
+            break;
+        case ELangGujarati:
+            locale = "gu_IN";
+            break;
+        case ELangHebrew:
+            locale = "he_IL";
+            break;
+        case ELangHindi:
+            locale = "hi_IN";
+            break;
+        case ELangIndonesian:
+            locale = "id_ID";
+            break;
+        case ELangIrish:
+            locale = "ga_IE";
+            break;
+        case ELangKannada:
+            locale = "kn_IN";
+            break;
+        case ELangKazakh:
+            locale = "kk_KZ";
+            break;
+        case ELangKhmer:
+            locale = "km_KH";
+            break;
+        case ELangKorean:
+            locale = "ko_KR";
+            break;
+        case ELangLao:
+            locale = "lo_LA";
+            break;
+        case ELangLatvian:
+            locale = "lv_LV";
+            break;
+        case ELangLithuanian:
+            locale = "lt_LT";
+            break;
+        case ELangMacedonian:
+            locale = "mk_MK";
+            break;
+        case ELangMalay:
+            locale = "ms_MY";
+            break;
+        case ELangMalayalam:
+            locale = "ml_IN";
+            break;
+        case ELangMarathi:
+            locale = "mr_IN";
+            break;
+        case ELangMoldavian:
+            locale = "ro_MD";
+            break;
+        case ELangMongolian:
+            locale = "mn_MN";
+            break;
+        case ELangPunjabi:
+            locale = "pa_IN";
+            break;
+        case ELangRomanian:
+            locale = "ro_RO";
+            break;
+        case ELangSerbian:
+            locale = "sr_RS";
+            break;
+        case ELangSinhalese:
+            locale = "si_LK";
+            break;
+        case ELangSomali:
+            locale = "so_SO";
+            break;
+        case ELangSwahili:
+            locale = "sw_KE";
+            break;
+        case ELangTajik:
+            locale = "tg_TJ";
+            break;
+        case ELangTamil:
+            locale = "ta_IN";
+            break;
+        case ELangTelugu:
+            locale = "te_IN";
+            break;
+        case ELangTibetan:
+            locale = "bo_CN";
+            break;
+        case ELangTigrinya:
+            locale = "ti_ET";
+            break;
+        case ELangTurkmen:
+            locale = "tk_TM";
+            break;
+        case ELangUkrainian:
+            locale = "uk_UA";
+            break;
+        case ELangUrdu:
+            locale = "ur_PK";
+            break;
+        case ELangUzbek:
+            locale = "uz_UZ";
+            break;
+        case ELangVietnamese:
+            locale = "vi_VN";
+            break;
+        case ELangWelsh:
+            locale = "cy_GB";
+            break;
+        case ELangZulu:
+            locale = "zu_ZA";
+            break;
+        case ELangEnglish:
```

Comment:
I'm wondering if `en_GB` might be a closer match, given that there are also separate `ELangAmerican` and `ELangInternationalEnglish` values.

Reply:
Yep!